packages/manager/apps/dedicated/client/app/ip/components/list/firewall/game/ip-ip-firewall-game.controller.js

```diff
@@ -338,11 +341,15 @@ export default /* @ngInject */ function IpGameFirewallCtrl(
 
     // Check if ports are valid
     if (
+      !self.rule.ports.from ||
       self.rule.ports.from === undefined ||
+      !self.rule.ports.to ||
       self.rule.ports.to === undefined
```

Comment:
```suggestion
      !self.rule.ports.from ||
      !self.rule.ports.to
```
`!` manages also `undefined` value